manifest.yml

```diff
@@ -68,5 +68,7 @@ files:
     dynamic: true
   - file: docker-compose.yml
     dynamic: true
+  - file: vapor-entrypoint.sh
```

Comment:
This could probably be just `entrypoint.sh` unless that clashes with something.

Reply:
Changed it to `entrypoint.sh`. With a lot of different custom services it can be nice to name the entry points, but it can also be nice to put all of your services in their own subdirectories and this is just a starting place anyway so hardly a pain point at this scale!